src/_buttons.scss

```diff
@@ -0,0 +1,12 @@
+.form-btn {
+  margin: 0.7em 0 .2em 0;
+  padding: 0.2em 0;
+  font-size: 2.4rem;
+  text-transform: uppercase;
+  font-weight: bold;
+  border-radius: 10px;
+  border: none;
+  background-color: #ffe81f;
+  color:  #000000;
+  cursor: pointer;
```

Comment:
cursor, border oraz reset backgroundu możesz zrobić w jakimś globalnym pliku resetu, bo będziesz musiał to za każdym razem powielać.